gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/PlanServiceImpl.java

```diff
@@ -149,6 +153,10 @@ public class PlanServiceImpl extends AbstractService implements PlanService {
     @Autowired
     private TagsValidationService tagsValidationService;
 
+    @Lazy
+    @Autowired
+    private GroupRepository groupRepository;
```

Comment:
You should use the group service linked to the group and not the repository here.